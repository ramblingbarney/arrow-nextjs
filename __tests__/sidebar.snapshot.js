import { render } from "@testing-library/react";
import SideBar from "../app/components/sidebar";

describe("Page Snapshot", () => {
  it("renders homepage unchanged", () => {
    const { container } = render(<SideBar />);
    expect(container).toMatchSnapshot();
  });
});
