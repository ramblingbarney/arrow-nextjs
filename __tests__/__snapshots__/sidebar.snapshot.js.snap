// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Snapshot renders homepage unchanged 1`] = `
<div>
  <div
    class="sidebar"
  >
    <div
      class="HAMBURGER-ICON space-y-2"
      data-testid="hamburger-icon"
    >
      <span
        class="block h-0.5 w-8 animate-pulse bg-gray-600"
      />
      <span
        class="block h-0.5 w-8 animate-pulse bg-gray-600"
      />
      <span
        class="block h-0.5 w-8 animate-pulse bg-gray-600"
      />
    </div>
    <div
      class="hidden"
    >
      <ul
        class="flex flex-col items-center justify-between min-h-[250px]"
      >
        <li
          class="border-b border-gray-600 my-8 uppercase"
        >
          <button
            class="nav-text"
          >
            Turn Direction 
            Off
          </button>
        </li>
        <li
          class="border-b border-gray-600 my-8 uppercase"
        >
          <button
            class="nav-text"
          >
            Turn MouseMove 
            Off
          </button>
        </li>
        <li
          class="border-b border-gray-600 my-8 uppercase"
        >
          <button
            class="nav-text"
          >
            Turn MouseTime 
            Off
          </button>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
